src/components/posts/userPosts.js

```diff
@@ -0,0 +1,132 @@
+import "./userPosts.css";
+
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+// import Paginate from "../Paginate/Paginate";
+
+const UserPosts = () => {
+  const [order, setOrder] = useState(false);
+  const [posts, setPosts] = useState([]);
+  const [comments, setComments] = useState([]);
+  const [arePostsLoading, setArePoastLoading] = useState(false);
+  const [areCommentsLoading, setAreCommentsLoading] = useState(false);
+  const [page, setPage] = useState(1);
+  // const [currentPage, setCurrentPage] = useState("1");
+  // const [postsPerPage, setPostsPerPage] = useState("5");
+  const { userId } = useParams();
+
+  const toggleComments = (postId) => {
+    let newPosts = posts.map((post) =>
+      post.id === postId
+        ? { ...post, showComments: !post.showComments }
+        : { ...post, showComments: false }
+    );
+    setPosts(newPosts);
+  };
+
+  const loadPostComments = async (postId) => {
+    setAreCommentsLoading(true);
+    toggleComments(postId);
+    const response = await axios.get(
+      `https://jsonplaceholder.typicode.com/comments?postId=${postId}`
+    );
+    const ExData = response.data;
```

Comment:
always use the camel notations while naming the variables and always use the descriptive names